pkg/storage/stores/shipper/compactor/retention/util_test.go

```diff
@@ -66,7 +66,7 @@ var (
 					RowShards: 16,
 				},
 				{
-					From:       dayFromTime(start.Add(49 * time.Hour)),
+					From:       dayFromTime(start.Add(73 * time.Hour)),
```

Comment:
The schema start date for this and the previous one is too close with just a days gap. Based on when the tests run, when we add `24h` > duration  < `48h` to `start`, it could result in jumping ahead by 2 days and end up falling into this schema instead of the previous one. For example, if `start` is `9th Jun 2300 hours` and we add `28h` like [the test here](https://github.com/grafana/loki/blob/806d6a503e1746a158f332d055f5d9fbd8b28d1a/pkg/storage/stores/shipper/compactor/retention/index_test.go#L28), we would fall into `Config[2]` as opposed to what the test expects i.e `Config[1]`.